### ISSUE TITLE
Update GeoTIFF code to use Apache Commons Imaging Alpha-3 API

### ISSUE THREAD

**Issue by gwlucastrig** [gwlucastrig/gridfour#21]
Last week, the Apache Commons Imaging project released a new version of their software library.  The new version features a number of improvements, but it also changes some of the methods used by the Gridfour applications that access data from GeoTIFF files. Therefore, I will be updating the Gridfour code to use the new API.

I have already been using version Alpha 3 to access elevation data from NASA's Shuttle Radar Topography Mission (SRTM) and other sources.  The new API is much cleaner when it comes to specifying data-read options such as accessing a subset of a product.  

I am also working on a demonstration application with accompanying web article that describes how to access metadata from a GeoTIFF.  I've used this approach with Gridfour to create data products that were geographically referenced and suitable for display in Google Earth and mainstream Geographic Information System (GIS) programs.

**Comment by gwlucastrig** [comment]
I've updated the code.  The Gridfour demo module now uses Commons Imaging Alpha 3. You can find the new version of the pom.xml under the "demo" folder.  The changes affect the Cloud-Optimized GeoTIFF demo.

I am working on a few more demonstration applications.

**Comment by gwlucastrig** [comment]
Gridfour release 1.0.2 uses Commons Imaging Alpha-3

You may include references to Imaging Alpha-3 in a Maven pom.xml specification using the following
```
<!-- https://mvnrepository.com/artifact/org.apache.commons/commons-imaging -->
<dependency>
    <groupId>org.apache.commons</groupId>
    <artifactId>commons-imaging</artifactId>
    <version>1.0-alpha3</version>
</dependency>
```

If you wish to download the imaging compiled object code in a Java Jar file manually, you may do so by visiting https://mvnrepository.com/artifact/org.apache.commons/commons-imaging/1.0-alpha3

You may obtain code for Commons Imaging at their Github page 
https://github.com/apache/commons-imaging

**Comment by gwlucastrig** [comment]
Changes for this issue were included in the Gridfour 1.0.2 release.  This issue is now closed.  Thanks.
